devops/scripts/benchmarks/benches/compute.py

```diff
@@ -698,3 +703,134 @@ def bin_args(self) -> list[str]:
             f"--ioq={self.ioq}",
             f"--ctrBasedEvents={self.ctrBasedEvents}",
         ]
+
+
+class UsmMemoryAllocation(ComputeBenchmark):
+    def __init__(self, bench, runtime: RUNTIMES, usm_memory_placement, size, measure_mode):
+        self.runtime = runtime
+        self.usm_memory_placement = usm_memory_placement
+        self.size = size
+        self.measure_mode = measure_mode
+        super().__init__(
+            bench, f"api_overhead_benchmark_{runtime.value}", "UsmMemoryAllocation"
+        )
+
+    def get_tags(self):
+        return [runtime_to_tag_name(self.runtime), "micro", "latency", "memory"]
+
+    def name(self):
+        return (
+            f"api_overhead_benchmark_{self.runtime.value} UsmMemoryAllocation "
+            f"usmMemoryPlacement:{self.usm_memory_placement} size:{self.size} measureMode:{self.measure_mode}"
+        )
+
+    def explicit_group(self):
+        return f"UsmMemoryAllocation {self.usm_memory_placement} {self.size} {self.measure_mode}"
```

Comment:
What are the results of all these benchmarks? Can't we use a single explicit group for all `UsmMemoryAllocation`?
The primary consideration is whether the results are within similar range, so that the bar charts look ok.